packages/compass-crud/package.json

```diff
@@ -91,7 +91,6 @@
     "electron": "^15.5.7",
     "enzyme": "^3.11.0",
     "eslint": "^7.25.0",
-    "fast-json-parse": "^1.0.3",
```

Comment:
Drive-by-fix; the package doesn’t provide any types, or any value at all really (“from Node 7 and forward this module is not useful anymore” in its README)